rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -205,10 +239,10 @@ def _maybe_enrich_curve_transfers(  # pylint: disable=no-self-use
             source_address == CURVE_Y_DEPOSIT and
             transaction.from_address == to_address
         ):
+            crypto_asset = event.asset.resolve_to_crypto_asset()
```

Comment:
why are you moving this higher? Just to fail faster?

Reply:
yes